roles/module_openapi_cloud/tasks/main.yaml

```
@@ -31,6 +31,12 @@
       - ( plugin['action'] == 'generate_ignore_files' ) or ( plugin['action'] == 'generate_all' )
       - plugin['name'] == 'vmware_rest'
 
+- name: Use autoflake to remove unused vars
+  ansible.builtin.command: "autoflake --in-place --remove-unused-variables -r {{ collection['path'] }}"
```

```suggestion
- name: Use autoflake to remove unused imports
  ansible.builtin.command: "autoflake --in-place --remove-all-unused-imports  -r {{ collection['path'] }}"
```

We don't have a problem with unused vars, at least not in vmware.vmware_rest ;-)

---

My bad! will fix this.